ES/ESdd036.xml

```diff
@@ -168,8 +168,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      <origPlace>
                         <placeName ref="INS0105DD">Dabra Dāmmo / DD</placeName>
                      </origPlace>
-                     <origDate notBefore="1700" notAfter="1799" cert="medium" evidence="lettering">Most likely sometime in the 18th century.</origDate>
-                     The manuscript consists of two production units <ref target="#p1"/> and <ref target="#p2"/>.  <ref target="#p1"/> contains two quires which were intended for a different manuscript. They serve as guard quires and 
+                     <origDate notBefore="1700" notAfter="1799" cert="medium" evidence="lettering">The manuscript was most likely produced sometime in the 18th century.</origDate>
+                     It consists of two production units <ref target="#p1"/> and <ref target="#p2"/>.  <ref target="#p1"/> contains two quires which were intended for a different manuscript. They serve as guard quires and 
```

Comment:
Please change the wording here; in the current version it looks like `<ref target="#p1"/>` is composed of a core AND two other quires intended for a different manuscript. I assume it is not qhat you mean.